TwoPaneLayout/library/src/main/java/com/microsoft/device/dualscreen/twopanelayout/TwoPaneLayoutImpl.kt

```diff
@@ -82,60 +80,82 @@ internal fun twoPaneMeasurePolicy(
             )
         }
 
-        when (layoutState) {
-            LayoutState.Fold -> { // single pane(screen), only one placeable for Fold
+        if (layoutState == LayoutState.Fold ||
+            (paneMode == TwoPaneMode.VerticalSingle && orientation == LayoutOrientation.Vertical) ||
+            (paneMode == TwoPaneMode.HorizontalSingle && orientation == LayoutOrientation.Horizontal)
+        ) { // single pane(screen), only one placeable for Fold
```

Comment:
Since the code of this condition is used two times (here and line `57`) maybe is better to extract the code and move it to its one function with a meaningful name so the code is not duplicated, and understanding this condition is easier.

e.g.

```
private fun isFoo() : Boolean {
       return if (layoutState == LayoutState.Fold ||
                      (paneMode == TwoPaneMode.VerticalSingle && orientation == LayoutOrientation.Vertical) ||
                      (paneMode == TwoPaneMode.HorizontalSingle && orientation == LayoutOrientation.Horizontal)
}
...
fun otherFunction () {

    if (isFoo()) {
            ......
    }
}

```

Reply:
Good idea. Updated